inc/Logging/QueryMonitor/RdbHttpRequestOutputHtml.php

```diff
@@ -0,0 +1,180 @@
+				$statuses = array_unique( array_column( $data->http, 'status_code' ) );
+
+				$this->before_tabular_output();
+
+				echo '<thead>';
+				echo '<tr>';
+				echo '<th scope="col">' . esc_html__( 'Method', 'remote-data-blocks' ) . '</th>';
+				echo '<th scope="col" class="qm-filterable-column">';
+				echo $this->build_filter( 'type', $statuses, __( 'Status', 'remote-data-blocks' ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo '</th>';
+				echo '<th scope="col" class="qm-filterable-column">';
+				echo $this->build_filter( 'host', $hostnames, __( 'URL', 'remote-data-blocks' ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo '</th>';
+				echo '<th scope="col" class="qm-filterable-column">';
+				echo $this->build_filter( 'result', $cache_statuses, __( 'Cache status', 'remote-data-blocks' ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo '</th>';
+				echo '<th scope="col">' . esc_html__( 'Caller', 'remote-data-blocks' ) . '</th>';
+				echo '</tr>';
+				echo '</thead>';
+
+				echo '<tbody>';
+
+				foreach ( $data->http as $row ) {
+					$row_attr = [];
+					$css = '';
+
+					if ( $row['error'] ) {
+						$css = 'qm-warn';
+					}
+
+					$uri = self::format_url( $row['uri'] );
+
+					$uri = preg_replace( '|^http:|', '<span class="qm-warn">http</span>:', $uri );
+
+					$stack = array();
+					$filtered_trace = $row['trace']->get_filtered_trace();
+
+					foreach ( $filtered_trace as $frame ) {
+						$stack[] = self::output_filename( $frame['display'], $frame['calling_file'], $frame['calling_line'] );
+					}
+
+					// These tokens are chosen to match the filters that ship in QM core.
+					$row_attr['data-qm-host'] = $row['hostname'];
+					$row_attr['data-qm-result'] = empty( $row['cache_status'] ) ? 'None' : $row['cache_status'];
+					$row_attr['data-qm-type'] = $row['status_code'];
+
+					$attr = '';
+					foreach ( $row_attr as $a => $v ) {
+						$attr .= ' ' . $a . '="' . esc_attr( (string) $v ) . '"';
+					}
+
+					printf(
+						'<tr %s class="%s">',
+						$attr, // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+						esc_attr( $css )
+					);
+
+					/* Method */
+					printf(
+						'<td>%s</td>',
+						esc_html( $row['method'] )
+					);
+
+					/* Status code */
+					printf(
+						'<td>%s</td>',
+						esc_html( $row['status_code'] )
+					);
+
+					/* URL */
+					printf(
+						'<td class="qm-url qm-ltr qm-wrap">%s</td>',
+						$uri // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+					);
+
+					/* Cache status */
+					echo '<td class="qm-has-toggle qm-ltr qm-nowrap">';
+					echo self::build_toggler(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+					echo '<ol>';
+					echo '<li>' . esc_html( $row['cache_status'] ) . '</li>';
+					echo '<div class="qm-toggled qm-info qm-supplemental">';
+
+					if ( '' !== $row['cache_age'] ) {
```

Comment:
Love this change. It makes that whole row so much more readable now, as you just need to click into it to see the relevant metadata like age, etc.